src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,4 +1,14 @@
-// Add the required types and props
-export const UserInfo = () => (
-  <>UserInfo markup</>
+import React from 'react';
+import { User } from '../../types/User';
+
+type Props = {
+  user: User,
+};
+
+export const UserInfo: React.FC<Props> = ({ user: { name, email } }) => (
+  <>
+    <a className="UserInfo" href={`mailto:${email}`}>
+      {name}
+    </a>
+  </>
```

Comment:
```suggestion
  <a className="UserInfo" href={`mailto:${email}`}>
    {name}
  </a>
```
No need for that in this case))